docs/Guides/CustomerVoice/SendReviewRequest.md

```diff
@@ -7,6 +7,8 @@ tags: [Review Request, Customer Voice]
 
 With Customer Voice, you can gather authentic reviews via email or text on the sites that matter most, to grow customer loyalty and boost sales.
 
+[Learn more about Customer Voice](https://support.vendasta.com/hc/en-us/sections/4419496276119-Getting-Started-Customer-Voice)
```

Comment:
Should this link to the parent page instead of the getting started page? 

```suggestion
[Learn more about Customer Voice](https://support.vendasta.com/hc/en-us/sections/4406957057687-Customer-Voice)
```

Reply:
Agreed, good catch